src/biogeochem/CNFireBaseMod.F90

```diff
@@ -329,8 +335,13 @@ subroutine CNFireFluxes (this, bounds, num_soilc, filter_soilc, num_soilp, filte
    integer                        , intent(in)    :: filter_soilc(:) ! filter for soil columns
    integer                        , intent(in)    :: num_soilp       ! number of soil patches in filter
    integer                        , intent(in)    :: filter_soilp(:) ! filter for soil patches
+   integer                        , intent(out)   :: num_actfirep    ! number of active patches on fire in filter
+   integer                        , intent(out)   :: filter_actfirep(:) ! filter for soil patches
+   integer                        , intent(out)   :: num_actfirec    ! number of active columns on fire in filter
+   integer                        , intent(out)   :: filter_actfirec(:) ! filter for soil columns
    type(dgvs_type)                , intent(inout) :: dgvs_inst
    type(cnveg_state_type)         , intent(inout) :: cnveg_state_inst
+   type(soilbiogeochem_carbonflux_type), intent(inout) :: soilbiogeochem_carbonflux_inst
```

Comment:
1. Please add a comment on the variable added and it's unit. 
2. Please align all :: in the declaration section.